content/id/docs/tasks/tools/kubeadm/high-availability.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Membuat Klaster dengan Ketersediaan Tinggi menggunakan kubeadm
+content_type: task
+weight: 60
+---
+
+<!-- overview -->
+
+Laman ini menjelaskan dua pendekatan yang berbeda untuk membuat klaster Kubernetes dengan ketersediaan tinggi menggunakan kubeadm:
+
+- Dengan Node _control plane_ yang bertumpuk (_stacked_). Pendekatan ini membutuhkan sumber daya infrastruktur yang lebih sedikit. Anggota-anggota etcd dan Node _control plane_ diletakkan pada tempat yang sama (_co-located_).
+- Dengan klaster etcd eksternal. Pendekatan ini membutuhkan lebih banyak sumber daya infrastruktur. Node _control plane_ dan anggota etcd berada pada tempat yang berbeda.
+
+Sebelum memulai, kamu harus memikirkan dengan matang pendekatan mana yang paling sesuai untuk kebutuhan aplikasi dan _environment_-mu. [Topik perbandingan berikut](/id/docs/setup/production-environment/tools/kubeadm/ha-topology/) menguraikan kelebihan dan kekurangan dari masing-masing pendekatan.
+
+Jika kamu menghadapi masalah dalam pembuatan klaster dengan ketersediaan tinggi, silakan berikan umpan balik
+pada [pelacak isu](https://github.com/kubernetes/kubeadm/issues/new) kubeadm.
+
+Lihat juga [dokumentasi pembaruan](/id/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-15).
+
+{{< caution >}}
+Laman ini tidak menunjukkan cara untuk menjalankan klastermu pada penyedia layanan _cloud_. Pada _environment cloud_, kedua pendekatan yang didokumentasikan di sini tidak akan bekerja untuk objek Service dengan tipe LoadBalancer maupun PersistentVolume dinamis.
+{{< /caution >}}
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+Untuk kedua metode kamu membutuhkan infrastruktur seperti berikut:
+
+- Tiga mesin yang memenuhi [kebutuhan minimum kubeadm](/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#before-you-begin) untuk
+  Node _master_
```

Comment:
we have replaced "master" with _control plane_, because it's kind of banned by tech community. I saw the rest of page using _control plane_ already
```suggestion
  Node _control plane_
```

Reply:
Ah, it's one of the non-inclusive words... Fixed in bc5c6c5.